rclcpp/test/rclcpp/test_timer.cpp

```diff
@@ -44,21 +51,28 @@ class TestTimer : public ::testing::Test
 
     test_node = std::make_shared<rclcpp::Node>("test_timer_node");
 
-    timer = test_node->create_wall_timer(
-      100ms,
-      [this]() -> void
-      {
+    auto lambda_fun = [this]() -> void {
```

Comment:
I would use a more explicit name

```suggestion
    auto timer_callback = [this]() -> void {
```

Reply:
Fixed